data/org.freedesktop.portal.Notification.xml

```diff
@@ -100,6 +100,32 @@
 
           There may be further restrictions on the supported kinds of icons.
 
+        * ``sound`` (``v``)
+
+          A serialized sound to add to the notification.
+          Supported sound formats are ogg/opus, ogg/vorbis and wav/pcm.
+
+          The format for
+          serialized sound is a tuple (sv) with the following supported keys:
+
+          * ``bytes`` (``ay``)
```

Comment:
I just considered to translate the `bytes` to a `file-descriptor` so that the backend impl only needs to handle fds. I was wondering if it makes even sense to have the `bytes `as an option here.

Reply:
I think it makes a lot of sense to only support fd passing here. For icons I might even add that bytes is deprecated to steer people away from it.

Reply:
+1 on only supporting fd passing.